### ISSUE TITLE
Potentially incorrect credible intervals on Bayesian Logistic Regression example notebook

### ISSUE THREAD

**Issue by rmill040** [pymc-devs/pymc#3329]
There seems to be an error in calculating a set of credible intervals in the [Bayesian Logistic Regression example](https://github.com/pymc-devs/pymc3/blob/master/docs/source/notebooks/GLM-logistic.ipynb). At cell 17, the code plots a histogram of the odds ratio:

```python
b = trace['educ']
plt.hist(np.exp(b), bins=20, normed=True)
plt.xlabel("Odds Ratio")
plt.show()
```
From the histogram, the x-axis scale of the odds ratio ranges from 1.37 to 1.43. In cell 18, the 95% credible interval is calculated as

```python
lb, ub = np.percentile(b, 2.5), np.percentile(b, 97.5)

print("P(%.3f < O.R. < %.3f) = 0.95"%(np.exp(3*lb),np.exp(3*ub)))
```
with an output of 

```
P(2.617 < O.R. < 2.824) = 0.95
```
These numbers are a bit high given the odds ratio range in the histogram. The error is multiplying the lb and ub by 3. Without rerunning the notebook, you can back out the correct credible intervals by taking the log, dividing by 3, and the exponentiating the result again as

```python
np.exp(np.log(2.617)/3) # 1.378
np.exp(np.log(2.824)/3) # 1.414
```

Now the credible intervals lie in the observed range of data with the new result
```
P(1.378 < O.R. < 1.414) = 0.95
```

Keep up the great work on this library!

**Comment by fonnesbeck** [comment]
It's not clear why the traces is being multiplied by 3 here. That is part of the problem.  Will take a look.

**Comment by springcoil** [comment]
I dunno why there's a multiple by 3, it must be a misunderstanding by myself. I think I wrote the original version of this. I think the solution is simply to remove the 3.

**Comment by springcoil** [comment]
@rmill040 If you wanted to open a quick PR - I'd be glad to merge this :)

**Comment by rmill040** [comment]
@springcoil No problem -- I'll get to this tonight!

**Comment by eigenfoo** [comment]
Closed by #3331
